src/credentials/sso/ssoSupport.ts

```diff
@@ -36,3 +36,16 @@ export function validateSsoProfile(profile: Profile, profileName: string): strin
 
     return undefined
 }
+
+export function isSsoProfile(profile: Profile): boolean {
+    return (
+        hasProfileProperty(profile, SSO_PROFILE_PROPERTY.SSO_START_URL) ||
+        hasProfileProperty(profile, SSO_PROFILE_PROPERTY.SSO_REGION) ||
+        hasProfileProperty(profile, SSO_PROFILE_PROPERTY.SSO_ACCOUNT_ID) ||
+        hasProfileProperty(profile, SSO_PROFILE_PROPERTY.SSO_ROLE_NAME)
```

Comment:
These four properties could probably be put into a constant array and used in both this function and validateSsoProfile